db/migrate/20160224175308_create_tools.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTools < ActiveRecord::Migration
+  def change
+    create_table :tools do |t|
+      t.string :title
+      t.text :description
+      t.integer :rents, default: 0
```

Comment:
Talked with Adrian about this before and we decided to start out with the easier solution and then build from there.